docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -82,7 +82,42 @@ Disable bootstrapping enrollment by setting `ZITI_BOOTSTRAP_ENROLLMENT=false` in
 
 ## Firewall
 
-The router's generated configuration always listens on a single, configurable TCP port (default: `3022`). You may set `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env` to bootstrap with a different port.
+The router's generated configuration always uses a single TCP port (default: `3022`) defined by `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env`.
+
+Confirm the router is running and listening on the expected port.
+
+```bash
+systemctl show -p MainPID --value ziti-router.service \
```

Comment:
isn't there any more straightforward way to do this? You've already declared the port up above. it would be simple to simply `netstat | grep {that port}` no? why all this complexity when a simple netstat would suffice? is there a reason?

Reply:
I can't think of a more straightforward to answer the question "Is the PID of the router service listening on any ports?" Checking for something listening on some port, even if that something is named "ziti" doesn't tell me if that ziti is the router service, or some other router container or native process.

Reply:
I explained the one-liner and added a simplified, though not authoritative listening port check: `sudo ss -tlnp | grep ziti`.